<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SeqEditor</class>
 <widget class="QDialog" name="SeqEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SeqEditor</string>
  </property>
  <widget class="QGroupBox" name="OverviewGroup">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>90</y>
     <width>361</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string>Overview</string>
   </property>
   <widget class="QPushButton" name="pbDelete">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>21</y>
      <width>241</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="AddProtocolGroup">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>390</y>
     <width>361</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Add Protocol</string>
   </property>
   <widget class="QPushButton" name="pbAddMeasure">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Measure</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbAddShelving">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Shelving</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leShelvingDelay">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>1000</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Shelving Delay</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ns</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Waiting Time</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>100</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ns</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leWaitingTime">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>461</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>560</y>
     <width>193</width>
     <height>28</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="AddPulseGroup">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>461</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>Add Pulse</string>
   </property>
   <widget class="QPushButton" name="pbAddPulse">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Pulse</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxChannel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>MicroWave</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Laser</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sig</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ref</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Channel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leStart">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leStop">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="leDuration">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Duration</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbAddScanStop">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>90</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Scan Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbAddScanStart">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Scan Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ns</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ns</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ns</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leStart_t">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="leStop_t">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="leDuration_t">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>120</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pbClearScan">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>120</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Scan</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="FileGroup">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>831</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>File Group</string>
   </property>
   <widget class="QLineEdit" name="leSeqName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>21</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="pbLoadSeq">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>20</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbSaveSeq">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>20</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
